R/breakpoints.R

```diff
@@ -1,7 +1,13 @@
 #' Define breakpoint values
 #'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
 #' A generic constructor for responsive breakpoints.
 #'
+#' @examples
+#' breakpoints(sm = c(4, 4, 4), md = c(3, 3, 6), lg = c(-2, 8, -2))
```

Comment:
Not important, but I usually prefer `@examples` to be the last roxygen section

Reply:
lol I was wondering when we'd have this conversation 😜 

I prefer putting `@params` and `@return` closer to the function signature, since I usually want to look at those together. So I like putting `@examples` after the description and other sections.

But it's not a hill I would refuse to get off of, just my personal preference